schematools/contrib/django/models.py

```diff
@@ -570,3 +570,17 @@ def related_model(self):
         dataset_name, table_name, column_name = [to_snake_case(part)
                                                  for part in self.relation.split(":")]
         return apps.all_models[dataset_name][table_name]
+
+
+ACTIVE_PROFILES = None
```

Comment:
Zou het nog handig zijn om er caching met timeout van te maken? Alhoewel, zo heel vaak zullen de profielen wschl. ook weer niet veranderen.

Reply:
Dat dacht ik ook in de begin, maar nu profiles en datasets worden samen gerefreshed en dat brengt wat meet duidelijkheid.